cmd/ci-clean/main.go

```diff
@@ -66,51 +66,45 @@ func main() {
 
 func cleanup(metalAuthToken, metalProjectID string) error {
```

Comment:
Cleanup could take a context.Context and we could send `context.Background()` rather than passing nil into the cleanup API calls.